tests/typecheck/contrib/admin/test_options.yml

```diff
@@ -32,7 +35,7 @@
               ("group", {"fields": ("c",), "classes": ("a",), "description": "foo"}),
               (_("lazy"), {"fields": ["bar"]})
           ]
-          form = ModelForm
+          form = TestModelForm
```

Comment:
This was incorrect - it should take a subclass of `ModelForm` - and it's now picked up 🎉